File: SDTideEventCell.m
//
//  SDTideEventCell.m
//  ShralpTide2
//
//  Created by Michael Parlee on 9/21/13.
//
//

#import "SDTideEventCell.h"

@implementation SDTideEventCell

- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
{
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
        self.timeLabel.adjustsFontSizeToFitWidth = YES;
        self.heightLabel.adjustsFontSizeToFitWidth = YES;
        self.typeLabel.adjustsFontSizeToFitWidth = YES;
    }
    return self;
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated
{
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: SDTideEventCell.swift
//
//  SDTideEventCell.swift
//  ShralpTide2
//
//  Created by Michael Parlee on 11/17/18.
//

import Foundation

@objc class SDTideEventCell: UITableViewCell {
    
    @IBOutlet weak var timeLabel: UILabel! {
        didSet {
            self.timeLabel.adjustsFontSizeToFitWidth = true
        }
    }
    @IBOutlet weak var heightLabel: UILabel! {
        didSet {
            self.heightLabel.adjustsFontSizeToFitWidth = true
        }
    }
    @IBOutlet weak var typeLabel: UILabel! {
        didSet {
            self.typeLabel.adjustsFontSizeToFitWidth = true
        }
    }
    
    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
        super.init(style: style, reuseIdentifier: reuseIdentifier)
    }
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
    }
}
